src/main/resources/channels-mongodb-1.7.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Why do we need 1.7? Or more generally speaking, why do we need more than one version to begin with? Is it for testing the channel stuff? If so, I'd say 1.8.0.Final should be "stable", and 1.9.0.Alpha1 should be "dev" (or whatever the name is).

Reply:
The current version in MCS is 1.5 and in quay we have images for 1.7, 1.8 and 1.9. Think about that rich customer that pays a lot of money to support ancient connector versions in the future. ;)
The coding on this issue started where 1.8 release was around the corner.
This will be cleaned up eventually when we prepare the real prod setup for summit. I want to wait until 1.9.0.Final is out. If you still want it removed I can do it.

Reply:
Ok, we can clean this up later on. One purpose of the managed service btw. is to exactly not keep supporting old versions ;)